### DISCUSSION TITLE
Can xmake replace pkg-config?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [xmake-io/xmake#5139] [Ideas]
Now

Xmake ≈ Make/Ninja + CMake/Meson + Vcpkg/Conan + distcc + ccache/sccache

Can it

Xmake ≈ Make/Ninja + CMake/Meson + Vcpkg/Conan + distcc + ccache/sccache + pkg-config? :smile:

**Comment by waruqi** [comment]
https://xrepo.xmake.io/#/getting_started?id=find-the-library-information-of-the-package

**Reply by Freed-Wu** [reply]
Oh, sorry. I means if xmake can recognize `*.pc` like pkg-config?

**Reply by waruqi** [reply]
That's not necessary.

**Reply by Freed-Wu** [reply]
Why it is unnecessary?
